tajo-core/src/main/java/org/apache/tajo/session/Session.java

```diff
@@ -121,6 +124,10 @@ public synchronized String getCurrentDatabase() {
     return currentDatabase;
   }
 
+  public synchronized void setQueryCache(LoadingCache<String, Expr> cache) { this.cache = cache; }
```

Comment:
The cache instance is always updated instead of adding a cache item to a singleton cache instance. I don't think that this is a valid cache operation.

In addition, please follow our coding convention.